src/EventTickets/resources/components/EventTicketsListItem.tsx

```diff
@@ -30,7 +30,7 @@ export default function EventTicketsListItem({ticket, currency, selectedTickets,
                     </button>
                 </div>
                 <p className={'givewp-event-tickets__tickets__ticket__quantity__availability'}>
-                    {ticket.quantity - selectedTickets} {__('remaining', 'give')}
+                    {ticketType.totalTickets - selectedTickets} {__('remaining', 'give')}
```

Comment:
Does `totalTickets` mean the total tickets allowed or the total tickets available?